src/ColAndreas.cpp

```diff
@@ -45,6 +45,7 @@ PLUGIN_EXPORT bool PLUGIN_CALL Load(void **ppData)
 	else
 	{
 		logprintf("No collision data found.");
+		logprintf("File /scriptfiles/colandreas/ColAndreas.cadb not exist.");
```

Comment:
I would rather you just replace the original line, that'd be better than saying the same thing twice :)